wp-content/themes/engage/templates/page-tool.twig

```diff
@@ -5,31 +5,51 @@
 
 		<div class="article__content">
 			<article class="quiz-creator-product-page" id="tool-landing-page">
```

Comment:
There are two `<article>` HTML elements defined on this page. We only need one. Probably just keep the top level one and change this one to a generic `<div>`